internal/pkg/log/assert.go

```diff
@@ -13,29 +13,27 @@ import (
 )
 
 // CompareJSONMessages checks that expected json messages appear in actual in the same order.
-// Actual string may have extra messages and the rest may have extra fields. String values are compared using wildcards.
+// Actual string may have extra debug and info messages and the rest may have extra fields. String values are compared using wildcards.
 // Returns nil if the expectations are met or an error with the first unmatched expected line and all remaining actual lines.
 func CompareJSONMessages(expected string, actual string) error {
```

Comment:
It logs tests, it can very easily happen that some important warning or error is skipped.

That's why I improved it, and warning/error messages must always be in expected.

In next commits, I fixed the tests where such an error occurred.